src/main/java/com/fasterxml/jackson/databind/jsontype/impl/AsDeductionTypeDeserializer.java

```diff
@@ -94,6 +103,37 @@ protected Map<BitSet, String> buildFingerprints(DeserializationConfig config, Co
         return fingerprints;
     }
 
+    private List<String> _findAliases(DeserializationConfig config, BeanPropertyDefinition property)
+    {
+        AnnotationIntrospector ai = config.getAnnotationIntrospector();
+        List<PropertyName> aliases = new ArrayList<>();
+        AnnotatedField field = property.getField();
```

Comment:
Should not be necessary to access all these; annotations should be merged by `POJOPropertyBuilder` (impl of `BeanPropertyDefinition`) so `prop.getPrimaryMember()` should be all that is needed.

Reply:
Oh, sweet. Thanks!